From 89d620021fbf0229fc609eaefc9311d848c8ab3b Mon Sep 17 00:00:00 2001
Message-ID: <89d620021fbf0229fc609eaefc9311d848c8ab3b.1698444193.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1698444193.git.sharlatanus@gmail.com>
References: <cover.1698444193.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 27 Oct 2023 01:17:38 +0100
Subject: [PATCH 11/19] gnu: python-astropy: Update to 5.3.4.

* gnu/packages/astronomy.scm (python-astropy): Update to 5.3.4.
[arguments]{phases}: Disable one failing test, reported upstream.
---
 gnu/packages/astronomy.scm | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 51053b1d58..937da94b03 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1412,13 +1412,13 @@ (define-public celestia-gtk
 (define-public python-astropy
   (package
     (name "python-astropy")
-    (version "5.3.3")
+    (version "5.3.4")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "astropy" version))
        (sha256
-        (base32 "1fwk7x4q1hgdf9m8q613c6q7045sam1g934vgqv588ksbngxyc03"))
+        (base32 "1n7iwvjari4xv37094cpiapmjhhm57b04hi4r40wqb5czbigg46l"))
        (modules '((guix build utils)))
        (snippet
         '(begin
@@ -1473,7 +1473,9 @@ (define-public python-astropy
                              "not remote_data"
                              ;; XXX: Check why this tests failing.
                              " and not test_ignore_sigint"
-                             " and not test_parquet_filter"))))))))
+                             " and not test_parquet_filter"
+                             ;; See https://github.com/astropy/astropy/issues/15537
+                             " and not test_pvstar"))))))))
     (native-inputs
      (list pkg-config
            python-colorlog
-- 
2.41.0

